app/helpers/lines_helper.rb

```diff
@@ -1,15 +1,13 @@
 module LinesHelper
   def lines
-    ENV['LINES'].present? ? split_and_strip(ENV['LINES']) : %w(DC MD VA)
+    LINES
   end
 
-  def current_line
+  def current_line_display
```

Comment:
My paranoia is satisfied that nothing will break. Smaller codebases are a nice change of pace!